adafruit_bno08x/__init__.py

```diff
@@ -482,32 +491,29 @@ class BNO08X:  # pylint: disable=too-many-instance-attributes, too-many-public-m
 
     """
 
-    def __init__(self, reset=None, debug=False):
-        self._debug = debug
-        self._reset = reset
+    def __init__(self, reset: Optional[dict] = None, debug: bool = False) -> None:
```

Comment:
I think `reset` will be `DigitalInOut` type instead of `dict`, it is still Optional though. Would need to change a few lines below here as well if so.